src/main/java/pl/tomaszdziurko/jvm_bloggers/utils/SyndFeedProducer.java

```diff
@@ -55,6 +50,6 @@
 
     public Optional<String> validUrlFromRss(String rss) {
         Optional<String> url = createFor(rss).map(SyndFeed::getLink);
-        return url.filter(URL_VALIDATOR::isValid);
+        return url.filter(Validators::isUrlValid);
```

Comment:
\[Checkstyle\] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4\.